ocraft-s2client-api/src/main/java/com/github/ocraft/s2client/api/controller/S2Controller.java

```diff
@@ -210,12 +210,7 @@ private S2Controller launch() {
         log.info("Launching Starcraft II with configuration: {}.", cfg);
         try {
             Path gameRoot = Paths.get(cfg.getString(GAME_EXE_ROOT));
-            String exeFile = gameRoot
-                    .resolve(Paths.get(
-                            ExecutableParser.VERSIONS_DIR,
-                            cfg.getString(GAME_EXE_BUILD),
-                            cfg.getString(GAME_EXE_FILE)))
-                    .toString();
+            String exeFile = cfg.getString(GAME_EXE_PATH);
```

Comment:
Currently ocraft is hardcoded to the newest sc2 version.

This change uses GAME_EXE_PATH config value, which first checks for an author's inputted path from setProcessPath() or from "-e=<processPath>" command line argument, and defaults to path of the newest version otherwise.